src/main/kotlin/network/warzone/mars/feature/Resource.kt

```diff
@@ -11,6 +12,7 @@ import java.util.*
 import kotlin.reflect.KClass
 
 sealed class ResourceType<out T : Feature<*>>(type: KClass<T>) {
+    object Achievements : ResourceType<AchievementFeature>(AchievementFeature::class)
```

Comment:
Please stick to singular nouns for the resource objects of this class

Reply:
Will do!